README.md

```diff
@@ -333,15 +376,47 @@ Source: [register.json](https:/github.com/edatasheets/digital-datasheets/blob/ma
 |bitFieldResetValue|Reset value of a bit field|String| |
 |bitFieldAccessType|Access type of a bit field|String| |
 
-####  4.5.22	 Specification Of Value
+####  4.5.25	 Description Of The Reliability Of a Component
```

Comment:
"Specification" instead of "Description" to be consistent.

Reply:
done